expr/func_aspercent_old_test.go

```diff
@@ -77,9 +77,9 @@ var b1AsPercentOfa1b1b1 = []schema.Point{
 }
 
 var b1AsPercentOfa1 = []schema.Point{
-	{Val: math.MaxFloat64, Ts: 10},
-	{Val: 100 * (-25.8) / 50.6, Ts: 20},
-	{Val: 100 * (math.MaxFloat64 / float64(1000)) / float64(1234567890), Ts: 30},
+	{Val: math.NaN(), Ts: 10},
```

Comment:
i can confirm. in graphite. 100/0 = None. technically, should be inf, but graphite has no concept for inf